test/parallel/test-v8-coverage.js

```diff
@@ -20,6 +20,18 @@ function nextdir() {
   const output = spawnSync(process.execPath, [
     require.resolve('../fixtures/v8-coverage/basic')
   ], { env: { ...process.env, NODE_V8_COVERAGE: coverageDirectory } });
+  try {
+    console.info('--- DEBUG ---');
```

Comment:
are the `console.info()` statements part of the test or purely informational? If informational, I'd prefer to omit them and use code comments instead. If they are part of the test, then please add a comment indicating that so that they are not accidentally removed later.

Reply:
there's a new build environment that fails for coverage, added since I last ran the suite. This is purely to see what the heck is going on.